contracts/ChildChainGaugeInjectoooor.sol

```diff
@@ -272,26 +288,27 @@ contract ChildChainGaugeInjector is ConfirmedOwner, Pausable, KeeperCompatibleIn
    */
     function sweep(address token) external onlyOwner {
         uint256 balance = IERC20(token).balanceOf(address(this));
```

Comment:
```suggestion
        uint256 balance = IERC20(token).balanceOf(address(this));
        address contractOwner = msg.sender;
```
and then use `contractOwner` below instead of calling `owner()`